src/client/Microsoft.Identity.Client/Internal/Logger/MsalLogger.cs

```diff
@@ -34,7 +33,7 @@ internal class MsalLogger : ICoreLogger
                     : " - " + correlationId;
             PiiLoggingEnabled = enablePiiLogging;
             _loggingCallback = loggingCallback;
-            _minLogLevel = logLevel;
+            LogLevel = logLevel;
```

Comment:
I don't think you need to make any changes to this class. Try to use the existing method `IsLoggingEnabled`

Reply:
ok good, updated